src/test/java/de/tum/in/www1/artemis/TeamIntegrationTest.java

```diff
@@ -91,7 +95,7 @@ void testTeamAssignmentConfig() {
         var teamAssignmentConfig = new TeamAssignmentConfig();
         teamAssignmentConfig.setExercise(exercise);
         assertThat(teamAssignmentConfig.getExercise()).isEqualTo(exercise);
-        System.out.println(teamAssignmentConfig);
+        log.debug(teamAssignmentConfig.toString());
```

Comment:
```suggestion
```
IMHO in this case it would be fine to just remove the output entirely. I don’t think it adds useful information in this small test method.